.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: Test
+
+on:
+  push:
+    branches: "master"
+  pull_request:
+    branches: "*"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go-version: [ 1.12.x, 1.13.x, 1.14.x ]
+        os: [ ubuntu-latest, macos-latest ]
+    env:
+      OS: ${{ matrix.os }}
+      GO_VERSION: ${{ matrix.go-version }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install nvim binary
+      uses: rhysd/action-setup-vim@v1
+      with:
+        neovim: true
+        version: nightly
+
+    - name: Test and vet
+      run: |
+        diff -u <(echo -n) <(gofmt -s -d .)
+        go vet ./...
```

Comment:
Should `go vet ./... &&` ?

Reply:
IIRC go vet will return exit 1 if failed. even if needs `&&` ?

Reply:
If `go vet is exit 1` and `go test` is exit 0, won't CI succeed?

Reply:
IIRC, yes